source/io_netcdf.F90

```diff
@@ -2126,7 +2117,7 @@ subroutine write_nstd_netcdf(data_file,field_id,            &
 
 !-----------------------------------------------------------------------
 !
-!  define start, count for all dimensions; do not allow out-of-bounds
+!  define sstart, count for all dimensions; do not allow out-of-bounds
```

Comment:
It looks like this was an inadvertent change? The variable defined below is named `start` so the original comment looks correct.